src/emailpro/emailpro.service.js

```diff
@@ -1197,5 +1197,17 @@ angular.module('Module.emailpro.services').service('EmailPro', [
         params: opts.params,
       });
     };
+
+    this.getLocalizedPrice = function (ovhSubsidiary, price, currencyCode) {
```

Comment:
The issue is still the same 😃 . If Italia wants to display only TTC too, then we would have to change this function and `emailpro-account-order.html`. See what I mean ? It would be more effective to have something like this :

`data-ng-if="ctrl.shouldDisplayOnlyAllTaxes"`

in the HTML file

Reply:
Got it! :)
I have made it more generic like you said, using a flag.